CHANGELOG.md

```diff
@@ -5,6 +5,8 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## Unpublished Changes
+## 10:00:0 08-05-2022
+* larva-patterns - Add `extended` variant for `pullquote` module.
```

Comment:
This need to be under `Unpublished Changes`

Reply:
Fixed it.